pkgs/tools/text/csvkit/default.nix

```diff
@@ -1,44 +1,33 @@
-{ lib, python3, glibcLocales }:
+{ lib, fetchpatch, python3 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "csvkit";
-  version = "1.0.4";
+  version = "1.0.5";
 
   src = python3.pkgs.fetchPypi {
     inherit pname version;
-    sha256 = "1830lb95rh1iyi3drlwxzb6y3pqkii0qiyzd40c1kvhvaf1s6lqk";
+    sha256 = "1ffmbzk4rxnl1yhqfl58v7kvl5m9cbvjm8v7xp4mvr00sgs91lvv";
   };
 
+  patches = [
```

Comment:
please add a comment as to why the patch is necesary, and when it's likely to be removed

Reply:
:+1:.